data/searchers/map planets.txt

```diff
@@ -77,9 +77,10 @@ planet Salus
 	landscape land/clouds_02
 	description `Salus is a very cloudy world, notable for its heat and the red tint that the cloud cover gives its sun. The largest producer of medical products of all Searcher planets, it is very rich, and boasts the only major military base in all of Searcher space. Your ship is scanned upon landing, and you are carefully watched to make sure you don't attempt to do anything illegal.`
 	spaceport `The spaceport on Salus is very large, dominated by the military base. There is a lot of traffic here, but it has managed to avoid the worst of the congestion by the use of an extraordinarily complicated transportation network. The landing pads are generously sized, and there are enough if them to land even the very largest of fleets. As soon as you landed, you were provided with a guide instructing you in the operation of the transportation network.`
+	security 1
 
 planet Tela
 	attributes searchers textiles
 	landscape land/mountain9
-	description `Tela is the largest cloth producer of Searcher space, despite its size. It is covered in massive textile mills, and under its surface are hidden thousands upon thousands of young Searchers who labour daily producing cloth. Tela's population never sees most of the cloth it produces again: it is shipped elsewhere, to where it can sell for a good deal more money. Because of this, Tela is a very poor world, whose inhabitants have to constantly struggle to survive.`
+	description `Tela is the largest cloth producer of Searcher space, despite its size. It is covered in massive textile mills, and under its surface are hidden thousands upon thousands of young Searchers who labor daily producing cloth. Tela's population never sees most of the cloth it produces again: it is shipped elsewhere, to where it can sell for a good deal of money. Because of this, Tela is a very poor world, whose inhabitants have to constantly struggle to survive.`
```

Comment:
The use of "labour" was actually intentional here. I do pretty much all of my work in Canadian English because it seems more natural to me.

But since this is aiming for vanilla, you're probably right that it needs to be switched.

```suggestion
	description `Tela is the largest cloth producer of Searcher space, despite its size. It is covered in massive textile mills, and under its surface are hidden thousands upon thousands of young Searchers who labor daily producing cloth. Tela's population never sees most of the cloth it produces again: it is shipped elsewhere, to where it can sell for a much higher price. Because of this, Tela is a very poor world, whose inhabitants have to constantly struggle to survive.`
```